### DISCUSSION TITLE
Invissible floor in Model Viewer

### DISCUSSION THREAD

**Discussion by forestgunn** [google/model-viewer#4519] [Show and tell]
How to create an invisible floor in Blender for Model Viewer?  Animation is needed to make objects appear from beneath the floor.

**Comment by danilokrulj** [comment]
I'm not sure if that's possible since the lowest point of the model gets placed on the ground plane. But maybe you will find a work around, good luck! :)

**Comment by forestgunn** [comment]
Only one solution for now - to scale  objects  from 0 to 100 in first 2 frames. They look like just appear from the floor level

**Reply by danilokrulj** [reply]
I thought about that but wasn't sure if it would work the way you wanted it to, but since I wasn't sure if it would work I didn't want to suggest it.
